packages/x-components/tests/e2e/cucumber/next-queries/next-queries.spec.ts

```diff
@@ -18,14 +20,22 @@ Given(
         }
       }
     };
-    cy.visit('/test/next-queries', {
-      qs: {
-        xModules: JSON.stringify(config)
-      }
-    });
   }
 );
 
+And('next queries response being mock {string}', (mockName: string) => {
+  cy.intercept('https://api.empathy.co/nextQueries', async req => {
+    const module = await import('./stubs/next-queries.stub');
+    req.reply(module[mockName as keyof typeof module]);
+  }).as('interceptedNextQueries');
```

Comment:
Why are you storing the next queries request in a variable? I can't find any usage of it.